react-components/src/tools/ToolRestrictions.js

```diff
@@ -8,21 +8,20 @@ import {
     FormCheckbox,
     FormNumberField,
     FormSelectField,
-    FormTextField,
     getMessage,
 } from "@cyverse-de/ui-lib";
-import { Field } from "formik";
+
 import { MenuItem, Typography } from "@material-ui/core";
+import { Field, getIn } from "formik";
+import numeral from "numeral";
 import PropTypes from "prop-types";
 
 const NETWORK_MODES = ["bridge", "none"];
 
 const ONE_GB = 1024 * 1024 * 1024;
 
-function convertToGB(size) {
-    let gbValue = Math.round(size / ONE_GB);
-    return gbValue + " GB";
-}
+const formatGBListItem = (size) => size && numeral(size).format("0 ib");
+const formatGBValue = (size) => size && numeral(size).format("0.0 ib");
```

Comment:
So simple! ✨ 
At least in the screenshots, the DE shows, for example, `GB` and Belphy will now show `GiB`.  Should we keep these units consistent to eliminate any confusion?  Not sure if one is actually calculating `GiB` and the other `GB`, or if the units are labeled incorrectly though.

Reply:
Good point. They're both actually calculating `GiB`, so I should probably update the GXT code to use that unit label as well.

Reply:
Is there a reason behind using `GiB` format instead of `GB` ?

Reply:
Honestly I haven't seen `GiB` formatting commonly used. And We still have data window showing file sizes in `GB` as far as I know.

Reply:
Just because I was curious: the data window seems to also use the binary equivalents. My file `/home/mian/analyses/gzip__for_pipeline__giant_disk_analysis1-2019-01-30-01-20-34.2/logs/docker-compose.yml` shows a size of 2469 bytes, which in KB would be about 2.47 and in KiB would be about 2.41. The data window shows "2.41 KB" (which is to say, we're labeling KiB as KB).

Reply:
(of course, some stuff just labels the binary ones without the `i`, but it seems like if we're more specific it's better :smile: )

Reply:
I've probably contributed to this in the DE 😊  I realize now I've had `*B` and `*iB` flipped in my head.

Reply:
Now am torn 🤔 should we change Data window to display `KiB` or update formatting here ...

Reply:
It's the unit that k8s is using for setting memory resource requirements: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-memory

I imagine it's also the units that Condor is using as well.

The `numeral` package will only calculate the correct binary value if using this unit.
Using this example again: 4,000,000,000 bytes is 4GB, but only 3.7GiB.

Reply:
https://github.com/cyverse-de/ui/blob/f437d7c55d19e35c80924f63f7144834bbc1c7ba/de-lib/src/main/java/org/iplantc/de/client/util/DiskResourceUtil.java#L334 - @ianmcorvidae you are right!

Reply:
Sounds good to me then. We can fix Data window to show sizes `iB`

Reply:
Yeah, condor uses the binary versions. https://htcondor.readthedocs.io/en/v8_9_2/man-pages/condor_submit.html#submit-description-file-commands (it, mildly annoyingly, doesn't let you actually say MiB or similar, it just treats M and MB as MiB).

My experience is that most people are more used to using/prefer the binary units, and most of the time things labeled KB/MB/GB/etc. are actually KiB/MiB/GiB/etc. I sorta wish they'd changed the unit names the other way, but I do also understand that K/M/G/T/etc. have pre-existing meanings based on powers of 10. So my thought is that using the binary units and labeling them accurately is the clearest way we can communicate, even if it might mean some users have to look up why exactly that `i` is in the unit.